_config.yml

```diff
@@ -1,4 +1,4 @@
-title: "Pod 22.PX.X"
+title: "Pod 23.APR.1"
```

Comment:
This is a duplicate change from #15.
You should either have it here, or just over there.

Reply:
Yes. sorry, I have removed it from this PR